src/modules/feature/board/board/BoardHeader.vue

```diff
@@ -214,4 +216,12 @@ watchEffect(() => {
 	font-family: var(--font-accent);
 	letter-spacing: $field-letter-spacing;
 }
+
+.board-header--item {
```

Comment:
```suggestion
.input-container {
```
I don't think you need to include "board-header" in the name because the styles are scoped to this component.